### PR TITLE
Add basic ETH RPC

### PR DESCRIPTION
**Pull request by torkelrogstad**
@octobocto  feel free to rip out the second commit, only for my testing purposes

### CODE REVIEW

**Comment by octobocto** [comment]
nice! think your commit is a good starting point

**Comment by torkelrogstad** [comment]
Seems like the RPC is made before I click submit? That's not good

**Comment by torkelrogstad** [comment]
Looks very nice, btw!

**Comment by octobocto** [comment]
> image
> ~Seems like the RPC is made before I click submit? That's not good~
> 
> Nvm, looks like a caching issue, where the response persisted after I typed more stuff. Could we do a list where each RPC invocation adds a new card to a scrollable list, which includes both the RPC made, and the response?

Sure!

File: lib/pages/tabs/ethereum/ethereum_rpc_tab_page.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  const EthereumRPCTabPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const SailPage(
+      scrollable: true,
+      title: 'Ethereum RPC',
+      subtitle:
+          'Here you can call eth rpcs directly to the eth-sidechain. Try typing in "eth_blockNumber" in the input below.',
```
[the remark this answers is not in the document]
Comment (octobocto) [reply lib/pages/tabs/ethereum/ethereum_rpc_tab_page.dart:20]: much better

**Pull request merged**